packages/devops/scripts/lambda/resources/startup.sh

```diff
@@ -1,7 +1,9 @@
-#!/bin/bash -le
+#!/bin/bash -leE
```

Comment:
The capital E means errors within called scripts bubble up to be caught by the top level `trap` (see below)